### PR TITLE
Fix: Fixed taskbar behavior in full screen mode

### LINKED ISSUE #10767: Bug: Taskbar doesn't un-hide when Files is full screened/maximized
### Description

My taskbar autohides. When every other app is open, if I hover to the bottom of my screen, the taskbar will show up. But this is not the case when this app is open. This doesn't let the taskbar show up. I have tried restarting the app, restarting the PC and even reinstalling the app. Yet, always, this app prevents the taskbar from showing up.

### Steps To Reproduce

1. Enable the option `Automatically hide the taskbar`
2. Open files app in Maximized Window and let taskbar hide
3. Hover to the bottom of the screen and try to open the taskbar (it won't show up)
4. Open any other app as maximized window and hover to the bottom of the screen and now, the taskbar will show up

For now, the only ways to show the taskbar is to press the windows key or close/minimize/restore the Files app

### Steps to fix

Reveal the taskbar on hover (reaching the bottom of the screen) when taskbar is set to autohide

### Files Version

Version: 2.4.2.0 OS Version: 10.0.22621.963

### Windows Version

Edition	Windows 11 Pro 
Version	22H2 
OS build	22621.963 
Experience	Windows Feature Experience Pack 1000.22638.1000.0

### Relevant Assets/Logs

None

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #10767...

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Toggle full screen mode
   2. Confirmed taskbar is show when hovering the cursor at the bottom of the screen 

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/MainWindow.xaml.cs [hunk 1]
```diff
@@ -27,6 +28,9 @@ public sealed partial class MainWindow : WindowEx
 
 		public IntPtr WindowHandle { get; }
 
+		[DllImport("user32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
+		static extern bool SetPropW(IntPtr hWnd, string lpString, IntPtr hData);
```
Comment (d2dyno1) [inline src/Files.App/MainWindow.xaml.cs:32]: Could you use Vanara's User32 class instead?

Comment (yaira2) [reply src/Files.App/MainWindow.xaml.cs:32]: Is there a benefit to that? I though adding more references to Vanara would make it harder to enable trimming.

Comment (d2dyno1) [reply src/Files.App/MainWindow.xaml.cs:32]: You've got a point. Could you move it then to `InteropHelpers`?

Comment (yaira2) [reply src/Files.App/MainWindow.xaml.cs]: Good suggestion, I addressed it in the last commit.

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**